src/ucp/wireup/address.c

```diff
@@ -491,12 +485,18 @@ ucp_address_pack_length(ucp_worker_h worker, void *ptr, size_t addr_length)
 
 static const void*
 ucp_address_unpack_length(ucp_worker_h worker, const void* flags_ptr, const void *ptr,
-                          size_t *addr_length, int is_ep_addr, int *is_last)
+                          size_t *addr_length, int is_ep_addr, int *is_last_iface)
 {
     ucp_rsc_index_t rsc_index;
     uct_iface_attr_t *attr;
     const ucp_address_unified_iface_attr_t *unified;
 
+    if (is_ep_addr) {
+        /* Caller should not use *is_last_iface for ep address, because for ep
+         * address last flag is part of lane index */
+        ucs_assert(is_last_iface == NULL);
```

Comment:
can be simplified:
```
/* Caller should not use *is_last_iface for ep address, because for address last flag is part of lane index */
ucs_assert(i!s_ep_addr || (is_last_iface == NULL));
```